### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

@jaydrogers Please consider using `typos`.

### CODE REVIEW

File: docs/components/content/Column.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <script setup>
 const props = defineProps({
     sticky: {
-        defualt: false
+        default: false
```
Comment (szepeviktor) [inline docs/components/content/Column.vue:13]: # 💄

**Comment by jaydrogers** [comment]
YES!!! Thank you @szepeviktor!

So good to hear from you again!

Hope you're doing well and thanks for suggesting all these fixes.

> Please consider using typos.

Is this a VScode plugin or a package of some sort?

> BTW Workflows are not PR friendly.

I know... I didn't realize that until recently 🙃

**Comment by szepeviktor** [comment]
> Hope you're doing well

I cannot buy work. Even not for money!

So I send random PR-s titled _Fix typos_

**Comment by szepeviktor** [comment]
https://github.com/crate-ci/typos 👁️‍🗨️

**Comment by jaydrogers** [comment]
Thanks for the suggestions. I am merging into my own branch so I can take it from here 🤝

It's great to hear from you again @szepeviktor!

**Pull request merged**